pkg/k8s/code/crud.go

```diff
@@ -0,0 +1,71 @@
+package code
```

Comment:
code?

Reply:
any other idea? 
it is the package that creates the remote synthing stateful set, and manages it

Reply:
moved to `/syncthing/k8s/`